DuggaSys/diagram.js

```diff
@@ -1558,13 +1558,20 @@ function boxSelect_Update(mouseX, mouseY)
 
         if (ctrlPressed) {
             var markedEntities = getElementsInsideCoordinateBox(rect);
-
             // Remove entity from previous context is the element is marked
             previousContext = previousContext.filter(entity => !markedEntities.includes(entity));
 
             context = [];
             context = context.concat(markedEntities);
             context = context.concat(previousContext);
+        }else if (altPressed) {
```

Comment:
Missing `var` declaration. This will lead to a global declaration which is unnecessary and even if we wanted to make a global we should mark them with the `global` keyword.

Reply:
I pressed the wrong line. I meant the line after.